src/main/java/com/radixdlt/engine/RadixEngine.java

```diff
@@ -118,6 +118,9 @@ public void store(Atom atom) throws RadixEngineException {
 
 		synchronized (stateUpdateEngineLock) {
 			stateCheckAndStore(atom, cmAtom);
+
+			// TODO update validator state here
```

Comment:
Left for a future PR.

Reply:
RE doesn't know about "validators". Comment should say something like "TODO return updated state for some given query"

Reply:
Ah, true that, will fix